src/sortable.c

```diff
@@ -170,6 +175,109 @@ RSSortingVector *SortingVector_RdbLoad(RedisModuleIO *rdb, int encver) {
   return vec;
 }
 
+// We know these won't change
```

Comment:
Actually no, we're merging this with RSValue in the aggregation branch. but okay for now

Reply:
I meant that as opposed to the constants in the header file, mapping them to our 'serialization' constants won't actually change; so there is an extra layer of translation to guard against the changing enum values.